docs/playbooks/oneclick-predict.md

```diff
@@ -24,6 +24,9 @@ All timings are estimates, see the individual pages for each of these playbooks
 - `MAS_INSTANCE_ID` Declare the instance ID for the MAS install
 - `MAS_CONFIG_DIR` Directory where generated config files will be saved (you may also provide pre-generated config files here)
 - `MAS_ENTITLEMENT_KEY` Your IBM Entitlement key to access the IBM Container Registry
+- `WML_INSTANCE_ID` Set Default value to "openshift"
+- `WML_URL` Set Default value to "https://internal-nginx-svc.ibm-cpd.svc:12443" . ibm-cpd in the URL corresponds to the project name (namespace) of cp4d installation 
+- `WML_VERSION` Set Default value to "4.5". The wml_version for cp4d 4.0.x will be 4.0, if cp4d is 4.5.x , wml_version should change to 4.5
```

Comment:
Instead of requiring user to provide WML_VERSION and CPD_VERSION ... based on the above, can you add code to calculate wml_version automatically.

e.g. 

- if cpd_version starts with 4.0 wml_version = 4.0
- if cpd_version starts with 4.5 wml_version = 4.5

Reply:
to unblock predict_fvt, can we get this merged as user provided?. i will create another PR and automate WML_VERSION later this week

Reply:
@lokesh-sreedhara @durera  pls review my changes

Reply:
@alequint thank you. the changes looks good to me. nice work!